test/helpers/kubectl.go

```diff
@@ -1959,11 +1959,22 @@ func (kub *Kubectl) WaitForKubeDNSEntry(serviceName, serviceNamespace string) er
 // WaitCleanAllTerminatingPods waits until all nodes that are in `Terminating`
 // state are deleted correctly in the platform. In case of excedding the
 // given timeout (in seconds) it returns an error
+
 func (kub *Kubectl) WaitCleanAllTerminatingPods(timeout time.Duration) error {
+	return kub.WaitCleanAllTerminatingPodsInNs("--all-namespaces", timeout)
+}
+
+// WaitCleanAllTerminatingPodsInNs waits until all nodes that are in `Terminating`
+// state are deleted correctly in the platform. In case of excedding the
+// given timeout (in seconds) it returns an error
+func (kub *Kubectl) WaitCleanAllTerminatingPodsInNs(ns string, timeout time.Duration) error {
 	body := func() bool {
+		if ns != "--all-namespaces" {
```

Comment:
In k8s libraries it is used to defined an empty string to be all namespaces. This condition could be done differently for consistency:
```
if ns == "" {
   ns = "--all-namespaces"
}else{
   ns = "-n "+ ns
}
```